fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/DimensionColumn.java

```diff
@@ -17,13 +17,23 @@
 
     /**
      * Constructor.
-     * Uses dimension DruidName for column name.
+     * Uses dimension api name for column name.
      *
-     * @param dimension  The column name
-     * @param physicalName  Physical column name backing dimension
+     * @param dimension  The column's corresponding dimension
      */
-    protected DimensionColumn(@NotNull Dimension dimension, @NotNull String physicalName) {
-        super(physicalName);
+    public DimensionColumn(@NotNull Dimension dimension) {
```

Comment:
This should be `protected` I think, since we still want to force people to use the `static` factory method, right?